src/schedule-and-details/credit-section/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+
+export const CREDIT_REQUIREMENTS_TYPE = {
+  grade: 'grade',
+  proctoredExam: 'proctoredExam',
+  reverification: 'reverification',
+};
+
+const CreditSection = ({ intl, creditRequirements }) => {
+  const requirementsInfo = [
+    {
+      id: CREDIT_REQUIREMENTS_TYPE.grade,
+      label: `${intl.formatMessage(messages.creditMinimumGrade)}`,
+      values: creditRequirements?.grade,
+      prefix: 'credit-minimum-grade',
+    },
+    {
+      id: CREDIT_REQUIREMENTS_TYPE.proctoredExam,
+      label: `${intl.formatMessage(messages.creditProctoredExam)}`,
+      values: creditRequirements?.proctoredExam,
+      prefix: 'credit-proctored-exam',
+    },
+    {
+      id: CREDIT_REQUIREMENTS_TYPE.reverification,
+      label: `${intl.formatMessage(messages.creditVerification)}`,
+      values: creditRequirements?.reverification,
+      prefix: 'credit-reverification',
+    },
+  ];
+
+  const renderRequirementsInfo = (requirement) => (
+    <li className="d-grid" key={requirement.id}>
+      <h4 htmlFor={requirement.prefix} className="text-gray-700">
```

Comment:
I think `htmlFor` is only used for the `<label>` HTML tag

Reply:
removed